Package.swift

```diff
@@ -8,17 +8,15 @@ import CompilerPluginSupport
 
 let availabilityTags: [_Availability] = [
     _Availability("FoundationPreview"), // Default FoundationPreview availability,
-    _Availability("FoundationPredicate", availability: .macOS14_0), // Predicate relies on pack parameter runtime support
-    _Availability("FoundationPredicateRegex", availability: .macOS15_0) // Predicate regexes rely on new stdlib APIs
+    _Availability("FoundationPredicate"), // Predicate relies on pack parameter runtime support
+    _Availability("FoundationPredicateRegex") // Predicate regexes rely on new stdlib APIs
```

Comment:
Do we still want keep these since now they just resolve to base availability?

Reply:
I think for now (to avoid a mass refactor) we can keep them but note that we'll come back to this at a later time to remove them and replace them with the "real" availability